src/tools/create-app-tester.js

```diff
@@ -36,8 +36,9 @@ const promisifyHandler = handler => {
 };
 
 // A shorthand compatible wrapper for testing.
-const createAppTester = (appRaw, { customStoreKey, skipHttpPatch } = {}) => {
-  const handler = createLambdaHandler(appRaw, { skipHttpPatch });
+const createAppTester = (appRaw, { customStoreKey } = {}) => {
+  const opts = appRaw.appFlags;
+  const handler = createLambdaHandler(appRaw, opts);
```

Comment:
So this will opt out of logging HTTP requests (even `z.request`), right? Though, it would be strange to `skipHttpPatch` _and_ still use `z.request`.

Reply:
`z.request` logs separately, so this'll opt out of logging made by anything _besides_ `z.request`. namely, 3rd party SDKs